hbase-server/src/main/java/org/apache/hadoop/hbase/quotas/SpaceQuotaRefresherChore.java

```diff
@@ -74,6 +75,11 @@ public SpaceQuotaRefresherChore(RegionServerSpaceQuotaManager manager, Connectio
   @Override
   protected void chore() {
     try {
+      // check whether Quota table is present or not.
+      if (!checkQuotaTableExists()) {
```

Comment:
This will result in a META read. This can be done only once in the life time of an RS right if we found Quota table exists? A Boolean state can help here.
Why do you think we should do a WARN log? It can be just INFO at max IMO.

Reply:
Hi @anoopsjohn , Thanks for the review. Added a boolean state to avoid the Meta read again.